File: public/fetch.php
<?php
$url = 'https://upload.wikimedia.org/wikipedia/commons/thumb/a/ab/Logo_of_East_Java.svg/200px-Logo_of_East_Java.svg.png';
$context = stream_context_create([
    'http' => [
        'header' => "User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64)\r\n"
    ]
]);
$data = file_get_contents($url, false, $context);
file_put_contents(__DIR__ . '/images/logo_jatim.png', $data);
echo "OK!";

File: public/fetch.py
import urllib.request
req = urllib.request.Request('https://upload.wikimedia.org/wikipedia/commons/thumb/a/ab/Logo_of_East_Java.svg/150px-Logo_of_East_Java.svg.png', headers={'User-Agent': 'Mozilla/5.0'})
response = urllib.request.urlopen(req)
open('d:/laragon/www/dashboard/public/images/logo_jatim.png', 'wb').write(response.read())
